src/js/common/schemaform/review/ReviewCollapsibleChapter.jsx

```diff
@@ -65,11 +65,14 @@ export default class ReviewCollapsibleChapter extends React.Component {
 
   render() {
     let pageContent = null;
+
     if (this.state.open) {
+      const { data, pages } = this.props;
+
       pageContent = (
         <div id={`collapsible-${this.id}`} className="usa-accordion-content">
-          {this.props.pages.map(page => {
-            const editing = this.props.data[page.pageKey].editMode;
+          {getActivePages(pages, data).map(page => {
```

Comment:
Can you add a test for this component where a conditional page is hidden?

I haven't written any tests for FormPage yet, so I can cover the changes there in the unit test item I'm working on.